Code/Tests/shared/src/test/scala/tests/rescala/static/events/filter_EventTest.scala

```diff
@@ -0,0 +1,22 @@
+package tests.rescala.static.events
```

Comment:
move all the tests you added into their own package (tests.rescala.property maybe)

Reply:
Done in https://github.com/rescala-lang/REScala/pull/32/commits/d4acd5adbe96b909ebca82d30a678e1fe94327dc